README.md

```diff
@@ -1 +1,55 @@
-# orm-mapping-db-to-ruby-object-lab
+# ORM Mapping Database Rows to Ruby Objects
+
+## Objectives
+
+1. Write a method that turns database rows into Ruby objects 
+2. Write methods that allow you to search a database
+
+## Overview
+
+We are going to continue building out the `Student` class of our app. We are in the middle of building an app to help a public school's adminstrators keep track of their students. In this app we have a `Student` class. Each individual students has two attributes, a `name` and a `grade`, i.e. 9th, 10th, 11th, etc. 
+
+Our administrators need to save the student objects that this class produces to a database. We've set up the database for you in the `config/environment.rb` file. 
+
+Let's take a look:
+
+```ruby
+require 'sqlite3'
+require_relative '../lib/student'
+
+DB = {:conn => SQLite3::Database.new("db/students.db")}
+```
+Your connection to the database can be referred to, throughout your program, like this: `DB[:conn]`. 
+
+So far we built the ability to create the students table in the database (`Student.create_table`), drop that table (`Student.drop_table`), and `#save` a student to the database.
+
+Now, we will need to create a method that takes a row from the database and turns it back into a Student object. We will call this `#new_from_db`.
+
+Next, we want to build a couple of methods to get information from the database. We will call these `.find_by_name` and `.all`. 
+
+Let's get started!
+
+## Instructions
+
+### The `#new_from_db` Method
+
+This is an instance method that accepts a row from the database as an argument. It then creates a new student object based on the information in the row. Remember, our database doesn't store Ruby objects, so we have to manually convert it ourself.
```

Comment:
new from db is an instance method? it feel like more of a class method because you want to be able to do:

```ruby
bob = Ruby.new_from_db("stuff")
```
right?

not

```ruby
bob = Ruby.new
bob.new_from_db
```